lib/bibdata_rs/config.toml

```diff
@@ -1,2 +1,2 @@
 [env]
-FIGGY_URL = "https://figgy-staging.princeton.edu/catalog.json?f%5Bephemera_project_ssim%5D%5B%5D=Born+Digital+Monographs%2C+Serials%2C+%26+Series+Reports&f%5Bhuman_readable_type_ssim%5D%5B%5D=Ephemera+Folder&f%5Bstate_ssim%5D%5B%5D=complete&per_page=100&q="
\ No newline at end of file
+FIGGY_BORN_DIGITAL_EPHEMERA_URL = "https://figgy-staging.princeton.edu/catalog.json?f%5Bephemera_project_ssim%5D%5B%5D=Born+Digital+Monographs%2C+Serials%2C+%26+Series+Reports&f%5Bhuman_readable_type_ssim%5D%5B%5D=Ephemera+Folder&f%5Bstate_ssim%5D%5B%5D=complete&per_page=100&q="
```

Comment:
@kevinreiss thanks you for the review. Per your request I updated the url to FIGGY_BORN_DIGITAL_EPHEMERA